packages/world/src/modules/core/CoreModule.sol

```diff
@@ -108,20 +108,20 @@ contract CoreModule is Module {
   }
 
   /**
-   * @notice Register the core systems in the World.
+   * @notice Register the internal systems in the World.
    */
-  function _registerCoreSystems() internal {
-    _registerSystem(accessManagementSystem, ACCESS_MANAGEMENT_SYSTEM_ID);
-    _registerSystem(balanceTransferSystem, BALANCE_TRANSFER_SYSTEM_ID);
-    _registerSystem(batchCallSystem, BATCH_CALL_SYSTEM_ID);
-    _registerSystem(coreRegistrationSystem, CORE_REGISTRATION_SYSTEM_ID);
+  function _registerInternalSystems() internal {
```

Comment:
feels a little bit off to call these "internal systems" - with that name I'd think of internal functions, ie something that's not externally accessible, but these systems become part of the "core" API 🙈

Reply:
since we're _in_ the `CoreModule` (soon `InitModule`), maybe we can just name these `_registerTables`, `_registerSystems`, and `_registerSystem`?

Reply:
went ahead and did this in https://github.com/latticexyz/mud/pull/2225/commits/e58b76fb6940d59327a763b09b847481b40f3b32